tests/conftest.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
not sure why this is important some sort of config? is this normal? looks like helpers? Do we have a place to put test helpers?